packages/@webex/plugin-meetings/src/multistream/sendSlotManager.ts

```diff
@@ -7,10 +7,20 @@ import {
   StreamState,
 } from '@webex/internal-media-core';
 
+/**
+ * This class is used to manage the sendSlots for the given media types.
+ */
 export default class SendSlotManager {
   private readonly slots: Map<MediaType, SendSlot> = new Map();
   private readonly LoggerProxy: any;
+  private sourceStateOverride: Map<MediaType, StreamState | null> = new Map();
```

Comment:
A few minor things:

1. Can this be `readonly`? Only the contents of the map I think are being changed; the map itself is never replaced.
2. Can this be renamed to `sourceStateOverrides` (since there is more than one, similar to `slots`).
3. I noticed the value can be set to `null` here, yet...